src/transport_performance/osm/osm_utils.py

```diff
@@ -53,6 +53,9 @@ def filter_osm(
 
     """
     # defence
+    _type_defence(pbf_pth, "pbf_pth", (pathlib.Path, str))
+    # if pbf_pth is str, convert to Path in order to access as_posix method
+    pbf_pth = pathlib.Path(pbf_pth)
```

Comment:
I think this was discussed a long time ago and I can't remember how did we agree to solve it. If user inputs a string using forward slashes, `pathlib.Path(str)` will fail. Probably not something to fix here but worth remembering.

```shell
>>>> a = 'path\asdf'
>>>> pathlib.Path(a)
PosixPath('path\x07sdf')
```